index.js

```diff
@@ -1,11 +1,13 @@
 import express from 'express';
 import session from 'express-session';
+import swaggerUi from 'swagger-ui-express';
 import cors from 'cors';
 import errorhandler from 'errorhandler';
 import ENV from 'dotenv';
 import morgan from 'morgan';
 import passport from 'passport';
 import routes from './routes';
+import documentation from './swagger.json';
```

Comment:
`./swagger` When you said this, at first you are importing `index.js` from a swagger directory when the import misses that file, it searches a file named swagger.js on the root, then after missing the file, it searches swagger.extension so we don't know the order of the extension so that can say that it will reach to .json easily. My feedback is to check when you are not referencing an index file from a certain child directory or a name.js file in the root directory. it's better to specify the extension of the file. `./swagger.json instead of ./swagger`